packages/cxl-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@conversionxl/cxl-ui",
-  "version": "1.0.0",
+  "version": "0.2.2",
```

Comment:
Has this been rebased? Why is version going backwards.

Reply:
Since this PR seems to be in a systemic error state, I'll stop review here, until next push hopefully sends correct code @dominik-stypula-polcode

Reply:
This was rebased, changes in version were applied later during the `masonry layout` experiments which I'm not familiar with:
https://github.com/dominik-stypula-polcode/aybolit/commit/7241dd53605a97fbb0e041f5e74750e4dee9fa7d - seems the history of commits is fine - isn't it?

Reply:
> seems the history of commits is fine

It's clearly not fine, as indicated by this version revert. Some old commits are inappropriately re-applied.